src/coreclr/src/inc/winwrap.h

```diff
@@ -57,7 +57,6 @@
 // winbase.h
 #undef GetBinaryType
 #undef GetShortPathName
```

Comment:
I am surprised that GetShortPathName is used for anything. Is it really needed?

Reply:
Actually, while it is called from `src/coreclr/src/tools/util/consoleargs.cpp`, GetCanonFilePath, I've just found that that method is not compiled for Unix, so we can remove that one as well.